config/webpack.dev.config.js

```diff
@@ -9,6 +9,7 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const path = require('path');
 const PostCssAutoprefixerPlugin = require('autoprefixer');
 const PostCssRTLCSS = require('postcss-rtlcss');
+const CustomMediaCSS = require('postcss-custom-media');
```

Comment:
nit: might recommend including `PostCss` as a prefix to be consistent with these other PostCSS plugin imports above.